tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -939,5 +940,68 @@ public function testGetFrameworks(): void
         $this->assertArrayHasKey('adapters', $framework);
     }
 
+    public function testConsoleAvailabilityEndpoint(): void
+    {
+        $site = $this->createSite([
+            'buildRuntime' => 'ssr-22',
+            'fallbackFile' => null,
+            'framework' => 'other',
+            'name' => 'Test Site',
+            'subdomain' => 'test-site',
+            'outputDirectory' => './',
+            'siteId' => ID::unique()
+        ]);
+
+        $siteId = $site['body']['$id'] ?? '';
+
+        $this->assertEquals(201, $site['headers']['status-code']);
+        $this->assertNotEmpty($site['body']['$id']);
+        $this->assertEquals('Test Site', $site['body']['name']);
+
+        $sitesDomain = System::getEnv('_APP_DOMAIN_SITES', '');
+        $domain = "test-site.{$sitesDomain}";
+        $ruleId1 = \md5($domain);
```

Comment:
This logic could change in future, we shouldn't relay on it.
We can listRules for site we just created, and get rule ID that way.